test/test.js

```diff
@@ -162,13 +149,15 @@ try {
     if (sentry.length) logOnDetails.sha_sentryfile = sentry;
 }
 catch (beef){
-    util.log("Cannae load the sentry. " + beef);
+    util.log("Cannot load the sentry. " + beef);
```

Comment:
Make a special pull request with returning of the "Cannae".

Don't let your dreams be dreams. Yesterday, you said tomorrow. So JUST DO IT. MAKE YOUR DREAMS COME TRUE.

```Kappa```